Week3/morejava.js

```diff
@@ -0,0 +1,181 @@
+//Create a function that takes 3 arguments and returns the sum of the these arguments.
+function sum(a, b, c) {
+    return a + b + c;
+}
+console.log(sum(1, 2, 3));
+
+//Create a function named colorCar that receives a color, and prints out, 'a red car' for example.
+function colorCar(x) {
+    return 'a ' + x + ' car';
+}
+console.log(colorCar('red'))
+
+//Create an object and a function that takes the object as a parameter and prints out all of its properties and values.
+let funObject = {
+    name: 'Lama',
+    age: 30,
+    city: 'Amsterdam'
+}
+
+function printOut(z) {
+    console.log(z)
+}
+printOut(funObject);
+
+//Create a function named vehicleType that receives a color,
+//and a code, 1 for car, 2 for motorbike.
+//And prints 'a blue motorbike' for example when called as vehicleType("blue", 2)
+function vehicleType(color, code) {
+    if (code === 1) {
+        console.log('a ' + color + ' car');
+    }
+    if (code === 2) {
+        console.log('a ' + color + ' motorbike');
+    }
+}
+vehicleType("blue", 2)
+vehicleType("red", 1)
+
+
+//Can you write the following without the if statement,
+//but with just as a single line with console.log(...);?
+
+if (3 === 3) {
+    console.log("yes");
+} else {
+    console.log("no");
+}
+
+console.log((3 === 3) ? 'yes' : 'no');
+
+//Create a function called vehicle, like before, 
+//but takes another parameter called age, so that vehicle("blue", 1, 5) prints 'a blue used car'
+
+function vehicle(color, code, years) {
+    if (code === 1 && years < 2) {
+        console.log('a ' + color + ' new car')
+
```

Comment:
Why so many spaces here in between? It doesn't look nice 🙁

Reply:
It is clean now, I think:)